spotbugs/src/main/java/edu/umd/cs/findbugs/detect/RuntimeExceptionCapture.java

```diff
@@ -200,8 +216,20 @@ public void visit(CodeException obj) {
                             System.out.println("Dead exception store at " + first);
                         }
                         caughtException.dead = true;
-                        break;
                     }
+
+                    // See if the excpetion is rethrown at the end of the handler.
+                    while (block != null) {
+                        InstructionHandle last = block.getLastInstruction();
+                        if (last != null) {
+                            if (last.getInstruction().getOpcode() == Const.ATHROW) {
```

Comment:
This could be combined with the previous line.

Reply:
These two lines got combined in https://github.com/spotbugs/spotbugs/pull/2112/commits/7b0d8382230ded583db874c4f0f0900a5f8505c8, so the conversation can be resolved.